jumanji/environments/packing/bin_pack/env_test.py

```diff
@@ -38,70 +41,6 @@
 from jumanji.types import TimeStep
 
 
-@pytest.fixture
```

Comment:
Is there a way to add your additions in a way that doesn't show so many changes? (like at the end of the file so it looks like an addition rather than a delete and an addition of the same thing - it would make it easier to review and easier to merge back into jumanji if the team would like the new features

Reply:
The problem here is that the isn't just an addition but more of a refactor, since the tests were originally not in a class. I could try to restore the wy that file was before and add only my tests into it if you think that would be easier to review and merge.

Reply:
Yes I don't think we should be changing other people's jumanji code if possible

Reply:
There still seems to be a lot of code removed - have you pushed your latest version here?